bcda/api/v1/api_test.go

```diff
@@ -126,17 +126,18 @@ func (s *APITestSuite) TestBulkEOBRequestNoQueue() {
 }
 
 func (s *APITestSuite) TestBulkPatientRequest() {
-	since := "2020-02-13T08:00:00.000-05:00"
-	bulkPatientRequestHelper("Patient", "", s)
-	s.TearDownTest()
-	s.SetupTest()
-	bulkPatientRequestHelper("Group/all", "", s)
-	s.TearDownTest()
-	s.SetupTest()
-	bulkPatientRequestHelper("Patient", since, s)
-	s.TearDownTest()
-	s.SetupTest()
-	bulkPatientRequestHelper("Group/all", since, s)
+	for _, since := range []string{"2020-02-13T08:00:00.000-05:00", "2020-02-13T08:00:00.000+05:00"} {
+		bulkPatientRequestHelper("Patient", "", s)
```

Comment:
There are 4 tests being run here in a loop.  When the loop runs, only the third and fourth tests are different (since the first two tests simply test with no `_since` parameter).  Should the first two be pulled out of the loop?